torch/jit/__init__.py

```diff
@@ -47,7 +47,7 @@
 from torch.jit._serialization import save, load
 from torch.jit._fuser import optimized_execution, fuser, last_executed_optimized_graph
 
-from torch.jit.cuda import stream
+from torch.jit.cuda import Stream, Event
```

Comment:
Why are these exposed like this? So that they can be created in eager mode, outside of TorchScript?

Reply:
No, this is for using in Torchscript. If we don't expose it here in `torch/jit/__init__.py` , `Stream`/ `Event` are not recognized. Like the below Runtime error. 

```
RuntimeError: 
object has no attribute cuda:
  File "/data/users/nikithamalgi/pytorch/test/jit/test_cuda.py", line 87
        @torch.jit.script
        def event_default_args() -> bool:
            e = torch.jit.cuda.Event()
                ~~~~~~~~~~~~~~ <--- HERE
            return e is not None
```

Reply:
Hm, but if the goal is to allow eager code that uses CUDA to be scripted as is, we should figure out how to expose these classes as `torch.cuda.Event` and `torch.cuda.Stream`. 

One idea I have is to try to see if we can do this desugaring somehow in `CUDAPythonModuleValue`. I *think* it should be as simple as returning `ClassValues` for `torch.cuda.Event` and `torch.cuda.Stream`?

Reply:
Although aliasing didn't work inside `torch/jit/__init__.py` / `torch/cuda/__init__.py` files, I added `test.cuda.Stream` = `test.classes.cuda.Stream` in `test_cuda.py` and that worked.

Reply:
Sure, but that's not going to help users unless we can figure out how to automatically include/import this alias at the top of their code, right?

Reply:
Yeah, that's right. If I can somehow remove the `circular import error`, then the aliasing will work. I tried out a few things to avoid this circular import, but those didn't help either.

Reply:
This is now removed.